src/containers/Tenant/ObjectSummary/ObjectSummary.tsx

```diff
@@ -154,11 +159,26 @@ export function ObjectSummary({
 
         overview.push({
             label: i18n('summary.created'),
-            value: formatDateTime(CreateStep, ''),
+            value: formatDateTime(CreateStep),
         });
 
         const {PathDescription} = currentObjectData;
 
+        if (PathDescription?.TableStats) {
+            const {DataSize, RowCount} = PathDescription.TableStats;
+
+            overview.push(
+                {
+                    label: i18n('summary.data-size'),
+                    value: toFormattedSize(DataSize),
+                },
+                {
+                    label: i18n('summary.row-count'),
+                    value: formatNumber(RowCount),
+                },
+            );
+        }
```

Comment:
Add `Data size` and `Row count` fields for entities, which have these params

![Screenshot 2024-07-23 at 15 52 33](https://github.com/user-attachments/assets/37483fdc-2832-4fbe-8075-54368c3727f6)